components/script/dom/subtlecrypto.rs

```diff
@@ -317,18 +623,95 @@ impl SubtleCrypto {
     }
 
     /// <https://w3c.github.io/webcrypto/#aes-cbc-operations>
-    fn export_key_aes_cbc(&self, format: KeyFormat, key: &CryptoKey) -> Result<Vec<u8>, Error> {
+    fn import_key_aes_cbc(
+        &self,
+        format: KeyFormat,
+        data: &[u8],
+        extractable: bool,
+        usages: Vec<KeyUsage>,
+    ) -> Result<DomRoot<CryptoKey>, Error> {
+        if usages.iter().any(|usage| {
```

Comment:
It would be nice to add the spec steps as comments here (and elsewhere). Makes it much easier to update the code when the specification changes in the future. Something like
```rs
// Step 1. If usages contains an entry which is not one of "encrypt", "decrypt", 
// "wrapKey" or "unwrapKey", then throw a SyntaxError.
``` 
in this case.